src/pages/car/Car.js

```diff
@@ -1,5 +1,33 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import getCars from '../../services/car.service';
 
-const Car = () => <div>Show single Car</div>;
+const Car = () => {
+  const dispatch = useDispatch();
+  const { carsList } = useSelector((state) => state.cars);
+  const isCarsStored = useSelector((state) => state.cars.isCarsStored);
```

Comment:
[optional] These two lines can be done at once by destructuring isCarsStored as well
 const { carsList, isCarsStored } = useSelector((state) => state.cars);

Reply:
@Fikerte-T Thanks for the notice I'll make sure to implement it.